scss/components/_typography.scss

```diff
@@ -243,6 +243,11 @@ p {
     line-height: $subheader-lineheight;
     color: $subheader-color;
   }
+
+  a { // To fix Outlook margin issue #421
+    Margin: default;
+    margin: default;
```

Comment:
Can you switch the order of the rules? Is `inherit` not what we want to use here?

Reply:
Order switched. I think we want `default`; we wouldn't want to inherit any margins from a parent element. At least that's the value that's used in #421, where I found this fix.